packages/cli/src/services/redis-client.service.ts

```diff
@@ -156,4 +190,33 @@ export class RedisClientService {
 				return { host, port: parseInt(port) };
 			});
 	}
+
+	@Debounce(1000)
+	emit<Event extends keyof RedisEventMap>(
+		event: Event,
+		...args: Array<RedisEventMap[Event]>
+	): boolean {
+		return super.emit(event, ...args);
+	}
+
+	private registerListeners() {
+		const { maxTimeout: maxTimeoutMs, retryInterval: retryIntervalMs } = this.config;
+
+		const retryInterval = Math.round(retryIntervalMs / 1000) + 's';
+		const maxTimeout = Math.round(maxTimeoutMs / 1000) + 's';
+
+		this.on('connection-lost', (cumulativeTimeoutMs) => {
+			const cumulativeTimeout = Math.round(cumulativeTimeoutMs / 1000) + 's';
+			const reconnectionMsg = `Trying to reconnect in ${retryInterval}...`;
```

Comment:
If we retry every 500ms, logging "reconnect in 1s" feels a bit imprecise. Could we perhaps format the values with one decimal? I.e. `0.5 s`? (also could move the formatting to a separate fn)

Reply:
Please see [891c1c7](https://github.com/n8n-io/n8n/pull/11007/commits/891c1c7bc8675dd61198c6ea689f1e8844735231)